src/libs/PolicyUtils.ts

```diff
@@ -184,14 +184,13 @@ function getTagListName(policyTags: OnyxEntry<PolicyTags>) {
 /**
  * Gets the tags of a policy for a specific key. Defaults to the first tag if no key is provided.
  */
-function getTagList(policyTags: OnyxCollection<PolicyTags>, tagKey: string) {
+function getTagList(policyTags: OnyxCollection<PolicyTags>, tagKey: string): PolicyTags {
     if (Object.keys(policyTags ?? {})?.length === 0) {
         return {};
     }
 
     const policyTagKey = tagKey ?? Object.keys(policyTags ?? {})[0];
-
-    return policyTags?.[policyTagKey]?.tags ?? {};
+    return (policyTags?.[policyTagKey]?.tags as unknown as PolicyTags) ?? {};
```

Comment:
[Update](https://github.com/Expensify/App/pull/34168/files#diff-4ec85ea72f25a41093d479be537473c4580bc7deb065882749f27e8eb0eb4f75) PolicyTag type instead
```suggestion
    return policyTags?.[policyTagKey]?.tags ?? {};
```